trustmanager/remoteks/server.go

```diff
@@ -0,0 +1,59 @@
+package remoteks
+
+import (
+	"strings"
+
+	"github.com/Sirupsen/logrus"
+	google_protobuf "github.com/golang/protobuf/ptypes/empty"
+	"golang.org/x/net/context"
+
+	"github.com/docker/notary/trustmanager"
+)
+
+// GRPCStorage is an implementer of the GRPC storage server. It passes through
+// the requested operations to an underlying trustmanager.Storage instance, translating
+// between the Go and GRPC interfaces.
+type GRPCStorage struct {
+	backend trustmanager.Storage
+}
+
+// NewGRPCStorage instantiates a new GRPC storage server using the provided
+// backend.
+func NewGRPCStorage(backend trustmanager.Storage) *GRPCStorage {
+	return &GRPCStorage{
+		backend: backend,
+	}
+}
+
+// Set writes the provided data under the given identifier.
+func (s *GRPCStorage) Set(ctx context.Context, msg *SetMsg) (*google_protobuf.Empty, error) {
+	logrus.Debugf("storing: %s", msg.FileName)
```

Comment:
Non-blocking:  Should the other RPC calls also be logged?  Or at least `Remove`, since both are writes?